### PR TITLE
Adds steps of scraper before data is actually used

### PR DESCRIPTION
**Pull request by csaunders**
This is just for parts of the scraping pipeline.

One part simply pulls the data down

Another part transforms the raw data into something that I think is useful.

Details:
1. From what I can see, **capsule_image** is the logo that doesn't change. **header_image** changes I'm pretty sure depending on whatever Steams marketing and design team decides I guess.
2. Instead of having to remember what app id we were working with we can just scrape it out of the page
3. We collect all the categories for a game, this means we can add more features later if we want. It also means that if a game supports single player and multi-player we'll know (though for our purpose this really doesn't matter).

Please review @carsonb @davefp

### CODE REVIEW

File: test/fixtures/steam_games.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  app_id: 1234
+  header_image: 'http://foo.com/abra.jpg'
+  capsule_image: 'http://foo.com/cadabra.jpg'
+  categories:
+    - 1
+  multi_player: true
+  updated_at: 2010-01-01
+
+cs_go:
+  title: "Counter Strike Source"
```
Comment (carsonb) [inline test/fixtures/steam_games.yml:19]: Should be GO.

**Comment by carsonb** [comment]
Grabbing all of the categories is a good idea. Perhaps one day Valve will identify those that use the Steam accounts for inviting, and we'd want to highlight these as "easier" to play with friends.

I don't have much feedback, seems fine. Can we set our own user agent for the page requests?

File: test/fixtures/steam_games.yml (hunk 1 above)
Comment (csaunders) [inline test/fixtures/steam_games.yml:19]: Wololololol
- Chris Saunders / sndrs.ca

**Comment by csaunders** [comment]
I believe we can. I think the default one is Ruby. Want a fake one or like be honest little devs?
- Chris Saunders / sndrs.ca

**Comment by carsonb** [comment]
Let's be honest, good little devs, but I want to include "your API should
offer this info" in the agent string.

**Comment by csaunders** [comment]
Hahahahaha.

`UserAgent: SteamSauna - Making the API you didn't`

**Comment by csaunders** [comment]
Added some remote tests and the UA string update.

**Comment by carsonb** [comment]
:penguin: :rocket:

**Pull request merged**